README.md

```diff
@@ -41,18 +41,20 @@ Finally, run NGINX by changing the working directory to the root directory of th
 nginx -c "$PWD/nginx.conf" -p "$PWD/dist/"
 ```
 
-We specify `-p "$PWD/dist/"` so that relative paths in [`nginx.conf`](/nginx.conf) are relative to `dist/`.
-
-NGINX effectively places ODK Central Frontend and ODK Central Backend at the same origin, avoiding cross-origin requests.
+NGINX effectively places ODK Central Frontend and ODK Central Backend at the same origin, avoiding cross-origin requests. We specify `-p "$PWD/dist/"` so that relative paths in [`nginx.conf`](/nginx.conf) are relative to `dist/`.
 
 ODK Central Frontend will be available on port 8989.
 
-Some ODK Central Frontend functionality requires HTTPS, for example, downloading files from ODK Central Backend. To access this functionality in development, one option is to use [`ngrok`](https://ngrok.com/download). By default, ODK Central Frontend is available on port 8989, so you can run `ngrok http 8989` to expose a temporary HTTPS URL that you can use. Some functionality additionally requires you to specify the HTTPS URL to ODK Central Backend, for example, getting a form in ODK Collect. To do so, set the `default.env.domain` property in [`config/default.json`](https://github.com/getodk/central-backend/blob/master/config/default.json) to the HTTPS URL, then restart the ODK Central Backend server if it is already running.
+ODK Central Frontend communicates with ODK Central Backend in part using a session cookie. The cookie is `Secure` and `HttpOnly`, but will be sent over HTTP on localhost. ODK Central Frontend also interacts directly or indirectly with data collection clients and with services:
+
+- To upload an XLSForm, run [pyxform-http](https://github.com/getodk/pyxform-http). ODK Central Frontend communicates with pyxform-http through ODK Central Backend.
+- You can use ODK Collect to scan an app user QR code and collect data. One option to do so is to use [`ngrok`](https://ngrok.com/download). ODK Central Frontend is available on port 8989, so you can run `ngrok http 8989` to expose a temporary HTTPS URL that you can use. You will also need to set the `default.env.domain` property in [`config/default.json`](https://github.com/getodk/central-backend/blob/master/config/default.json) to the HTTPS URL, then restart ODK Central Backend if it is already running.
```

Comment:
Does this work for making submissions? I vaguely remember it didn't.

Reply:
I just double-checked submitting from Collect and was able to do so! 📩

Reply:
🙏 That makes life a lot easier, then!